src/Evolution/Systems/Cce/Actions/ReceiveWorldtubeData.hpp

```diff
@@ -93,41 +96,6 @@ struct ReceiveWorldtubeData {
     // provided for return type inference. This line will never be executed.
     return std::forward_as_tuple(std::move(box));
   }
-
-  template <
-      typename DbTags, typename... InboxTags, typename ArrayIndex,
-      Requires<
-          tmpl::list_contains_v<
-              tmpl::list<InboxTags...>,
-              Cce::ReceiveTags::BoundaryData<
-                  typename Metavariables::cce_boundary_communication_tags>> and
-          tmpl::list_contains_v<DbTags, ::Tags::TimeStepId>> = nullptr>
-  static bool is_ready(
-      const db::DataBox<DbTags>& box,
-      const tuples::TaggedTuple<InboxTags...>& inboxes,
-      const Parallel::GlobalCache<Metavariables>& /*cache*/,
-      const ArrayIndex& /*array_index*/) noexcept {
-    return tuples::get<Cce::ReceiveTags::BoundaryData<
```

Comment:
Just use `if constexpr` here, that's much easier... Plus it seems you've changed the behavior of this code by trying to make it generic...